.github/labeler-issue.yml

```
@@ -0,0 +1,12 @@
+api:
+  - '(Devopness API)'
+website:
+  - '(Devopness website)'
+web-app:
+  - '(Devopness web app)'
+docs:
+  - '(Documentation)'
+sdk:
+  - '(SDK)'
```

As I added in my original suggestion [1], the criteria for applying the `sdk` label should not require parenthesis.

Imagine this: two issues are opened by two different users: one for `SDK-JS` and another one for `SDK-Python`. What do both issues have in common? They are related to SDK. Therefore both issues, even not selecting a specific `SDK` category should get the common label `sdk` and each one of them also having a second specific label, `sdk-javascript` or `sdk-python`.

Makes sense?
```suggestion
  - 'SDK'
```

[1] https://github.com/devopness/devopness/pull/811#discussion_r1414397713

---

I see. The issue-labeler's [README.md](https://github.com/github/issue-labeler/blob/master/README.md) had only given an example using parentheses so I thought they were supposed to be necessary. But upon inspecting the [source code](https://github.com/github/issue-labeler/blob/master/src/main.ts) I realized that wasn't the case. Parentheses removed.